hw-model/c-binding/build.rs

```diff
@@ -13,6 +13,8 @@ fn main() {
     let config = cbindgen::Config {
         header: Some(String::from_str("// Licensed under the Apache-2.0 license").unwrap()),
         language: cbindgen::Language::C,
+        pragma_once: true,
```

Comment:
nit: We might want to use the `include_guard` option instead since `#pragma once` is technically compiler-specific (although it's widely supported in gcc and clang for a long time) and we don't really know which compiler various hardware integrators are going to use.

Reply:
Cool, I didn't see that option initially